TESTING.md

```diff
@@ -57,18 +60,58 @@ Then you can run a test plan e.g. on F35 system.
 # tmt run -vvv prepare discover provision -h virtual -i Fedora-35 execute finish
 ```
 
-The above command will dispose the system when tests are finished.
+The above command will download Fedora-35 image and use it for a newly created virtual
+system. Also, due to `finish` command the system will disposde when tests are executed.
 However for debugging purposes you may want to access test system once
 tests are finished. The command below will give you a shell after all tests are finished.
 
 ```
 # cd keylime-tests
-# tmt run -vvv prepare discover provision -h virtual -i Fedora-35 execute login finish
+# tmt run -vvv prepare discover provision -h virtual -i Fedora-35 -c system execute login finish
 ```
 
 You can use it to inspect test logs or even modify test sources and run your tests
 manually following the method described above. For this purpose you can find test sources under `/var/tmp/tmt/run-*/packit-ci/discover/default/tests/`.
 
+### Running multi-host tests
+
+`tmt` cannot schedule multi-host tests yet. However, we can use `tmt` to do the necessary setup and then execute the test manually.
+
+For `Multihost/basic-attestation` test we need at least 3 test systems that will be started using `tmt`. Please, make sure your workstation is properly configured as described in the section above.
+
+Let's start with opening 3 terminals on your workstation.
+
+On terminal 1:
+```
+$ git clone https://github.com/RedHat-SP-Security/keylime-tests.git
+```
+
+In the next step we will start 3 virtual systems using `tmt`.
+
+In case you do not have Fedora-35 image already downloaded (in `/var/tmp/tmt/testcloud/images`) you may want to run commands below in the 1st terminal first to avoid simultaneous downloads.
+
+On all terminals:
+```
+$ cd keylime-tests
+$ tmt run -vvv prepare discover -h fmf -t 'configure_tpm_emulator' -t 'install_upstream_keylime' -t 'Multihost/basic-attestation' provision -h virtual -i Fedora-35 execute login finish
```

Comment:
`-c system` should be here as well, no?

Reply:
For singlehost tests it is not necessary. But maybe it would be better to keep the cmdline consistent. I will add it.

Reply:
But here we are talking about the multi-host ones, no? I see that we have `-c` in the previous section, which should be the one on singlehost tests. But I agree that we should keep it consistent, if it will not break things :)

Reply:
Aaah, you are right! Fixed now.